main.nf

```diff
@@ -18,19 +18,42 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/viralrecon --reads '*_R{1,2}.fastq.gz' -profile docker
+    nextflow run nf-core/viralrecon --input samplesheet.csv --genome hg38 -profile docker
 
     Mandatory arguments:
-      --reads [file]                Path to input data (must be surrounded with quotes)
-      -profile [str]                Configuration profile to use. Can use multiple (comma separated)
-                                    Available: conda, docker, singularity, test, awsbatch, <institute> and more
-
-    Options:
-      --genome [str]                  Name of iGenomes reference
-      --single_end [bool]             Specifies that the input is single-end reads
+      --input [file]                  Comma-separated file containing information about the samples in the experiment (see docs/usage.md)
+      --protocol [str]                Specifies the type of protocol used for sequencing i.e. "metagenomics" or "amplicon". (Default: "metagenomics")
+      --host_fasta [file]             Path to Fasta reference for host genome. Not mandatory when using reference in iGenomes config via --genome
```

Comment:
I would put the `--host_fasta` in the references

Reply:
Since I'm getting it's the one from iGenomes.
Maybe keep the `--fasta` for that one.
Don't know about it actually.

Reply:
Its a mandatory parameter which is why I put it there :)

Reply:
I'm guessing that references are actually mandatory parameters too